.eslintrc.js

```diff
@@ -1,26 +1,90 @@
 module.exports = {
     root: true,
-    plugins: ['@typescript-eslint', 'promise', 'import', 'jsdoc'],
     env: {
         node: true,
-        es6: true,
-        browser: true
+        browser: true,
+        es6: true
```

Comment:
There are some reorderings that are still not alphabetical, maybe sort by alphabet?